cookbooks/stateofthemap/templates/default/apache.jekyll.erb

```diff
@@ -14,7 +14,9 @@
 
 <VirtualHost *:443>
         ServerName <%= @year %>.stateofthemap.com
-        ServerAlias <%= @year %>.sotm.org
+        <% @aliases.each do |alias_name| -%>
+          ServerAlias <%= alias_name %>
+       <% end -%>
```

Comment:
What's the reason for moving the aliases out into the recipe? It seems inconsistent to construct the primary name here but not the aliases?

Also the indentation here is inconsistent.

Reply:
Updated to be consistent with alias usage.